static/sass/print.scss

```
@@ -91,7 +95,12 @@ footer,
 .global-nav__header-list,
 .dropdown-window,
 .dropdown-window-overlay,
-.p-navigation--secondary {
+.p-navigation--secondary,
+.p-sticky-toc {
+  display: none;
```

```suggestion
  display: none !important;
```

Let's make this trump all other display rules as we know for sure we don't need to override it anywhere else down the cascade.